src/jit/simd.cpp

```diff
@@ -437,15 +437,15 @@ var_types Compiler::getBaseTypeAndSizeOfSIMDType(CORINFO_CLASS_HANDLE typeHnd, u
             size         = Vector256SizeBytes;
             JITDUMP("  Known type Vector256<ulong>\n");
         }
-        else if (typeHnd == m_simdHandleCache->Vector256FloatHandle)
```

Comment:
It appears that you deleted the `Vector256FloatHandle` case?

Reply:
No, it is the first handle in the `Vector256` block, this was supposed to be the `Vector128` handle and it was coded wrong

Reply:
For reference: https://github.com/dotnet/coreclr/pull/21097/files#diff-02808e0f227d4d50d3268d1106cfbf16R380

Reply:
Ah, thanks.